src/main/java/us/icarenow/web/model/entity/User.java

```diff
@@ -0,0 +1,52 @@
+package us.icarenow.web.model.entity;
+
+import javax.persistence.*;
+import java.util.List;
+
+@Entity
+@Table(name="icn_user")
+public class User {
+
+    @Id
```

Comment:
please wait for Adi's changes. Once his changes are in main branch then update your branchand adapt to his changes